frontend/src/views/CellCards/components/CellCard/components/EnrichedGenesTable/index.tsx

```diff
@@ -0,0 +1,156 @@
+  const [selectedOrganism, setSelectedOrganism] = useState("");
+
+  const uniqueOrganisms = useMemo(() => {
+    if (!genes) return [];
+    const organisms = new Set<string>();
+    for (const markerGene of genes) {
+      organisms.add(markerGene.organism);
+    }
+    if (!selectedOrganism && Array.from(organisms).includes("Homo sapiens"))
+      setSelectedOrganism("Homo sapiens");
+    else if (!selectedOrganism)
+      setSelectedOrganism(Array.from(organisms).at(0) ?? "");
+
+    const organismsArray = Array.from(organisms).sort((a, b) => {
+      if (a === "Homo sapiens") return -1;
+      if (b === "Homo sapiens") return 1;
+      return a.localeCompare(b);
+    });
+    return organismsArray;
+  }, [genes, cellTypeId]);
+
+  const tableRows: TableRow[] = useMemo(() => {
+    if (!genes) return [];
+    const rows = [];
+    for (const markerGene of genes) {
+      const { pc, me, name, symbol, organism } = markerGene;
+      if (organism !== selectedOrganism) continue;
+      rows.push({
+        symbol: symbol,
+        name,
+        me: me.toFixed(2),
+        pc: (pc * 100).toFixed(1),
+      });
+    }
+    return rows;
+  }, [genes, selectedOrganism, cellTypeId]);
+
+  useEffect(() => {
+    return () => {
+      setSelectedOrganism("");
+    };
+  }, []);
+
+  const genesForShareUrl = tableRows.map((row) => row.symbol).join("%2C");
+
+  const handleChange = (event: SelectChangeEvent<unknown>) => {
+    setSelectedOrganism(event.target.value as string);
+  };
+
+  return (
+    <div data-testid={CELL_CARD_ENRICHED_GENES_TABLE}>
+      <TableTitleWrapper>
+        <TableTitleOuterWrapper>
+          <TableTitleInnerWrapper columnGap={4}>
+            <TableTitle>Highly Expressed Genes</TableTitle>
+            <HelpTooltip
+              text={
+                <>
+                  {
+                    "The marker genes listed below are computationally derived from the "
+                  }
+                  <Link label={"CELLxGENE corpus"} url={ROUTES.DATASETS} />
+                  {
+                    ". They are computed utilizing the same methodology as featured in our "
+                  }
+                  <Link
+                    label={
+                      "Find Marker Genes feature from the Gene Expression application"
+                    }
+                    url={ROUTES.FMG_DOCS}
+                  />
+                  {"."}
+                </>
+              }
+            />
+          </TableTitleInnerWrapper>
+          {tableRows.length > 0 && (
+            <TableTitleInnerWrapper>
+              <DropdownSelect
+                handleChange={handleChange}
+                options={uniqueOrganisms}
+                selectedOption={selectedOrganism}
+                testId={CELL_CARD_ENRICHED_GENES_TABLE_DROPDOWN}
+              />
+              <Link
+                url={`${ROUTES.WHERE_IS_MY_GENE}?genes=${genesForShareUrl}`}
+                label="Open in Gene Expression"
+              />
```

Comment:
I guess we don't need to, but do we want to use a share url version?

Reply:
Is this going to f up our analytics?

Reply:
We can probably append an additional `?internal-share=true` flag that doesn't trigger analytics